va/validation-authority.go

```diff
@@ -664,6 +664,7 @@ func (va *ValidationAuthorityImpl) PerformValidation(ctx context.Context, domain
 	}
 
 	va.stats.TimingDuration(fmt.Sprintf("VA.Validations.%s.%s", challenge.Type, resultStatus), time.Since(vStart), 1.0)
+	va.stats.Inc(fmt.Sprintf("VA.ValidationResults.%s.%s", challenge.Type, resultStatus), 1, 1.0)
```

Comment:
I would leave this out. The nature of statsd means that you can count validation TimingDuration events pretty easily.